phpBB/phpbb/install/console/command/converter/convert.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license       GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\install\console\command\converter;
+
+use phpbb\install\exception\installer_exception;
+use phpbb\install\converter\module\converter_obtain_data\module;
+use phpbb\install\converter\controller\helper;
+use phpbb\install\helper\config;
+use phpbb\install\helper\install_helper;
+use phpbb\install\helper\iohandler\cli_iohandler;
+use phpbb\install\helper\iohandler\factory;
+use phpbb\install\helper\container_factory;
+use phpbb\install\installer_configuration;
+use phpbb\language\language;
+use Symfony\Component\Config\Definition\Exception\Exception;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Yaml;
+
+class convert extends \phpbb\console\command\command
+{
+	/**
+	 * @var factory
+	 */
+	protected $iohandler_factory;
+
+	/**
+	 * @var install_helper
+	 */
+	protected $helper;
+
+	/**
+	 * @var language
+	 */
+	protected $language;
+
+	protected $obtain_data_module;
+
+	protected $container_factory;
+
+	protected $install_config;
+	/**
+	 * Constructor
+	 *
+	 * @param language			$language
+	 * @param factory			$factory
+	 * @param installer			$installer
+	 * @param install_helper	$install_helper
+	 */
+	public function __construct(config $install_config, container_factory $container, language $language, factory $factory, module $obtain_data,  helper $helper)
+	{
+		$this->iohandler_factory = $factory;
+		$this->install_config = $install_config;
+		$this->container_factory = $container;
+		$this->language = $language;
+		$this->obtain_data_module = $obtain_data;
+		$this->helper = $helper;
+
+		parent::__construct(new \phpbb\user($language, 'datetime'));
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function configure()
+	{
+		$this
+			->setName('convert')
+			->addArgument(
+				'config-file',
+				InputArgument::REQUIRED,
+				$this->language->lang('CLI_CONFIG_FILE'))
+			->setDescription($this->language->lang('CLI_CF_CONVERT'))
+		;
+	}
+
+	/**
+	 * Executes the command install.
+	 *
+	 * Install the board
+	 *
+	 * @param InputInterface  $input  An InputInterface instance
+	 * @param OutputInterface $output An OutputInterface instance
+	 *
+	 * @return null
+	 */
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		//ALWAYS PURGE CACHES ELSE DI CONTAINER WONT BE REFRESHED
```

Comment:
I will remove it once I generate documentation for this project. Till that time let it be there since its an important warning and kept me in a loop for a long time.